src/main/kotlin/world/player/item/banking/regularBank/openBank.kts

```diff
@@ -2,11 +2,65 @@ package world.player.item.banking.regularBank
 
 import api.predef.*
 import io.luna.game.event.impl.ServerStateChangedEvent.ServerLaunchEvent
+import io.luna.game.model.def.NpcDefinition
+import io.luna.game.model.mob.*
+
+val DEFAULT_BANKER = 494
+
+val bankerIds = mutableListOf<Int>()
 
-// Load all banking objects, make them open the bank.
 on(ServerLaunchEvent::class) {
+    // Find all banker npcs and store their ids
+    NpcDefinition.ALL
```

Comment:
Can you move this to Banking.kt, model it how the objects are done `Banking.loadBankingNpcs()`